src/components/property-list/index.js

```diff
@@ -30,12 +30,12 @@ function PropertyList({ properties }) {
             {data.map(([property, value]) => {
                 return (
                     <div className="property-wrapper" key={property}>
-                        <div>
```

Comment:
This too needed for layout
![BCD9CA5C-D2AD-443B-9A53-AAB60DB80BA8](https://user-images.githubusercontent.com/10927011/185236251-65b1cea9-d0bf-4163-a313-3b574eefb4e4.jpeg)